modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json

```diff
@@ -183,8 +183,8 @@
 	"group-id-confirmation": "Group ID Confirmation",
 	"hamina-finland": "Hamina, Finland",
 	"hide": "Hide",
-	"high-priority-contact-removed": "successfully added from",
 	"high-priority-contact-added": "successfully removed from",
+	"high-priority-contact-removed": "successfully added from",
```

Comment:
Same here, I think you've changed the translations here.

Reply:
DOne! thx